### DISCUSSION TITLE
SU2+pyamg installation problem

### DISCUSSION THREAD

**Discussion by Andre174201** [su2code/SU2#2172] [General]
Hy everybody, i followed all the instructions provided in the following https://github.com/su2code/SU2/issues/948.
I still cannot figure it out how to properly install the pyamg extension. Can anybody here provide the right instructions related to the installation? Thank you!!!!

**Comment by arsh-k** [comment]
Hey I am having the same problem with _amgio. Have you found a solution for this problem yet? If yes, it would be very helpful if you could share it with me.
